packages/yarnpkg-pnp/sources/esm-loader/hooks/load.ts

```diff
@@ -13,6 +13,9 @@ export async function load(
     importAssertions?: {
```

Comment:
Didn't Node recently drop support for import assertions? Should we also remove it, or is that something to postpone until the minimum supported is Node 22?

Reply:
They did, in v22 [as indicated here](https://github.com/yarnpkg/berry/pull/6268/files#diff-aa5a8675637cc8375ebd5258968d3034f215b1db3a92213a9746f59993e19848R22).

Since we try to match whatever the Node.js versions we support allows it needs to stay until the minimum supported is Node 22.
The code ignores `importAssertions` in >= v22.